bakery/src/scripts/checksum_single.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+exec 2> >(tee checksum-book/stderr >&2)
```

Comment:
checksum-book should probably not be checksum-book (should probably be some param)